### PR TITLE
PUBDEV-6300 Flows throws error on an empty path while performing 'Import Files' operation

### PR DESCRIPTION
**Pull request by Pscheidl**
JIRA: 
https://0xdata.atlassian.net/browse/PUBDEV-6300

Depends on changes in Flow:
https://github.com/h2oai/h2o-flow/pull/130

### CODE REVIEW

File: h2o-core/src/main/java/water/persist/PersistManager.java [hunk 1]
```diff
@@ -247,6 +247,11 @@ private static boolean httpUrlExists(String URLName){
    * @return List of matches
    */
   public List<String> calcTypeaheadMatches(String filter, int limit) {
+    filter = filter.trim();
```
Comment (Pscheidl) [inline h2o-core/src/main/java/water/persist/PersistManager.java]: Let's "calculate" with trimmed value anyway.

File: h2o-core/src/main/java/water/persist/PersistManager.java [hunk 2]
```diff
@@ -247,6 +247,11 @@ private static boolean httpUrlExists(String URLName){
...
+    if (filter.isEmpty()) {
+      return new ArrayList<>();
```
Comment (Pscheidl) [inline h2o-core/src/main/java/water/persist/PersistManager.java]: Default behavior is to return nothing if the path is non-existing. As whitespaces or empty path means non-existing, return nothing as well.

**Comment by Pscheidl** [comment]
Waiting for https://github.com/h2oai/h2o-flow/pull/130 to be accepted by more people, then releasing new version of Flow. Then `WIP` label will be removed.

File: h2o-core/src/main/java/water/persist/PersistManager.java (hunk 2 above)
Comment (honzasterba) [reply h2o-core/src/main/java/water/persist/PersistManager.java]: Collections.emptyList() is better

**Comment by Pscheidl** [comment]
@honzasterba Resolved.

**Pull request merged**